esp-alloc/src/macros.rs

```diff
@@ -1,24 +1,25 @@
 //! Macros provided for convenience
 
-/// Create a heap allocator providing a heap of the given size in bytes
-///
-/// You can only have ONE allocator at most
+/// Initialize a global heap allocator providing a heap of the given size in
+/// bytes
 #[macro_export]
 macro_rules! heap_allocator {
```

Comment:
I suppose there is no way to make this runtime configurable without pre-allocating some area in a linker script? It would be really nice to configure this via `Config` and let `esp_hal::init` do the heavy lifting instead of having to remember to call this.

Probably not something we need to worry about now though.

Reply:
probably not with esp-hal as that doesn't need alloc, but if we ever get to the point of a distribution crate, we can pass a big slice to it's config